IBAnimatable/FadeAnimator.swift

```diff
@@ -20,12 +20,9 @@ public class FadeAnimator: NSObject, AnimatedTransitioning {
     self.transitionDuration = transitionDuration
 
     switch direction {
-    case .in:
-      self.transitionAnimationType = .fade(direction: .in)
-      self.reverseAnimationType = .fade(direction: .out)
-    case .out:
-      self.transitionAnimationType = .fade(direction: .out)
-      self.reverseAnimationType = .fade(direction: .in)
+    case .in, .out:
+      self.transitionAnimationType = .fade(direction: direction)
+      self.reverseAnimationType = .fade(direction: direction.opposite)
     default:
```

Comment:
Is this switch still mandatory? 
If it's to be sure that we are using a compatible direction for a fade, I would be in favour to add a `fatalError`:

```swift
if direction != .in && direction != .out && direction != .cross {
   fatalError("Invalid direction for a fade transition")
}
```